multiuser_posix.go

```diff
@@ -153,11 +156,23 @@ func MkdirAllTaskUser(dir string, perms os.FileMode) (err error) {
 	return nil
 }
 
-func makeFileOrDirReadWritableForUser(recurse bool, dir string, user *gwruntime.OSUser) error {
+func makeFileOrDirReadWritableForUser(recurse bool, fileOrDir string, user *gwruntime.OSUser) error {
 	if recurse {
-		return host.Run("/usr/sbin/chown", "-R", user.Name+":staff", dir)
+		switch runtime.GOOS {
+		case "darwin":
+			return host.Run("/usr/sbin/chown", "-R", user.Name+":staff", fileOrDir)
```

Comment:
Is there a reason why we're not using the `os` package here? If so, let's document it.

https://golang.org/pkg/os/#Chown

Reply:
++ You are quite right - there was a reason we weren't able to use the `os` package implementation, and I should have documented why, because I have myself forgotten now! I will have a think about this and try to remember, or if I can't, I'll switch it to use `os` package, see why it breaks, switch it back and add an explanation why.